app/lib/services.rb

```diff
@@ -16,7 +16,7 @@ def self.cached_content_item(base_path)
   end
 
   def self.rummager
```

Comment:
should we rename these too?

Reply:
Good question! Maybe in another PR? This one is about fixing a warning to tidy up the test output. A future one could be to make the methods accurate.

I notice the search API have decided _not_ to do a find-and-replace because they want to avoid messing up `git blame`: https://github.com/alphagov/search-api#search-api

Reply:
Yeah this might be a bigger piece of work